accessing
failure: aPPFailure

	failure := aPPFailure